ui/.template-lintrc.js

```diff
@@ -12,4 +12,7 @@ module.exports = {
     },
     'no-implicit-this': { allow: ['keyboard-commands'] },
   },
+  ignore: [
+    'app/components/breadcrumbs/*' // using {{(modifier)}} syntax
+  ]
```

Comment:
It seems like `{{(modifier ...)}}` is valid conditional syntax, and the absence of `()` in fact breaks this functionality, but the template linter doesn't like it, so ignoring it in the breadcrumb files.

Reply:
`blocking`: I don't think the linter was able to pick up on this because the test suite failed to run.